EnrichmentFormSample/EnrichmentFormSample/Scripts/widgets/enrichmentFormWidget.js

```diff
@@ -0,0 +1,34 @@
+﻿angular.module('EnrichmentFormSample')
+    .controller('EnrichmentFormSample.WidgetController', ['$scope', 'platformWebApp.bladeNavigationService', function ($scope, bladeNavigationService) {
+        var blade = $scope.blade;
+        var propertyNames = ['City', 'Country', 'Position', 'State', 'StreetAddress', 'Zip', 'geolocation'];
```

Comment:
can "geolocation" be "Geolocation"? Same as others

Reply:
we don't need geolocation - Position is used instead - will be removed